public/app/features/org/all.ts

```diff
@@ -1,3 +1,4 @@
 import './SelectOrgCtrl';
 import './NewOrgCtrl';
 import './UserInviteCtrl';
+import './prefs_control';
```

Comment:
Can this file be named PrefControlCtrl to match the file names guide of the other files in this folder.

Reply:
Sure. Will fix